blog/relacja-z-meetupu-hitachi-karate/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Relacja z meetupu Hitachi Energy - edycja Karate'
+authors: niewiarowski
+date: '2023-09-23'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'meetcontent'
+  - 'spolecznosc'
+coverImage: '1693820333064.png'
+---
+
+Jako ekipa techwriterów i techwriterek z [Hitachi Energy](https://www.linkedin.com/company/hitachienergy/mycompany/) w składzie: Julia Dąbrowska, Mateusz Barszcz, Piotr Sroka, Monika Dąbrowska, Andrzej Niewiarowski oraz członkini honorowa - Patrycja Pyrek - przeprowadziliśmy wyjątkowy meetup, którego spoiwem była kultura Japonii. 
+
+<!--truncate-->
+
+## Fusion Meetup o Technical Writingu i Karate
+### Informacje o spotkaniu
```

Comment:
Nie wiem czy  mamy jakieś zasady dotyczące nie zbijania nagłówków jeden po drugim? 
MS Style guide mówi:
![image](https://github.com/techwriter-pl/techwriter-pl.github.io/assets/119131560/99468d8f-edd4-4583-9029-29da731b0716)

Może usunąć nagłówek 'Informacje o spotkaniu' i z paragraf z linijki 22 zmienić we wstęp do sekcji Fusion Meetup o Technical Writingu i Karate?

Reply:
jest to dobry pomysl, dwa naglowki obok siebie nie sa potrzebne.